src/drive.cpp

```diff
@@ -0,0 +1,38 @@
+#include "drive.hpp"
+#include "util.hpp"
+
+#include <cmath>
+
+using namespace sdss_robotics_c;
+
+Drive::Drive(std::vector <int> leftMotorPorts, std::vector <int> rightMotorPorts, pros::motor_gearset_e_t gearset, pros::motor_brake_mode_e_t brakeMode) {
+    for(int port : leftMotorPorts) {
+        pros::Motor motor(std::abs(port), ConvertGearset(gearset), IsReversed(port), pros::E_MOTOR_ENCODER_DEGREES);
+        motor.set_brake_mode(brakeMode);
```

Comment:
All pretty good.  Again, creating a class for all of this is useful as it's more intuitive and clear on what everything does and reduces repeating code.